test-app/src/main/java/org/readium/r2/testapp/shared/views/Preferences.kt

```diff
@@ -383,19 +382,26 @@ private fun Item(
         } else {
             Modifier
         },
-        text = {
-            val alpha = if (isActive) 1.0f else ContentAlpha.disabled
-            CompositionLocalProvider(LocalContentAlpha provides alpha) {
-                Text(title)
+        headlineContent = {
+            val alphaForText = if (isActive) 1.0f else 0.38f
```

Comment:
I can, it didn't use one before so I left as is.

Reply:
It's probably an oversight. It's more maintainable to use `Group` instead of directly setting the alpha, for when Material 4 will inevitably change the emphasis rules.

Reply:
Done.